common/networkpolicies/base/model-registry.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: model-registry
+  namespace: kubeflow
+spec:
+  podSelector:
+    matchExpressions:
+      - key: component
+        operator: In
+        values:
+          - model-registry-server
+  ingress:
+    - from:
+        - namespaceSelector:
+            matchExpressions:
+              - key: app.kubernetes.io/part-of
+                operator: In
+                values:
+                  - kubeflow-profile
+        - namespaceSelector:
+            matchExpressions:
+              - key: kubernetes.io/metadata.name
+                operator: In
+                values:
+                  - istio-system
+        - podSelector: {} # allow all pods from the same namespace
```

Comment:
I think this one here is already the default networkpolicy allow-same-namespace or so. It does not belong into the one for the model registry.

Reply:
since this line is removed as requested with https://github.com/kubeflow/manifests/commit/5871263a7fe2f3c8403f9f2ca914bba874bcb752 resolving this thread.